### ISSUE TITLE
documentation generator: source code window should be (at least) 80em wide if the browser width allows it

### ISSUE THREAD

**Issue by inoas** [gleam-lang/gleam#1804]
Right now example code inlined via ```gleam is hard to read even if it breaks at 80 chars usually.

**Comment by lpil** [comment]
Sorry, could you expand a little on what you mean?

**Comment by inoas** [comment]
the source code "divs" should be max width of the screen size but not larger than 80 or 100em... so that we don't get horizontal scrolling in source code examples embedded in markdown.

**Comment by lpil** [comment]
Ah, so you're proposing we make the page a little wider to fit 80 columns in code blocks?

**Comment by inoas** [comment]
Different options:

- Add another breakpoint / make the whole page wider
- depending on breakpoint make just source code rendering wider so it is at least 80 columns

Is 80 columns the desired target for Gleam code?

All I need to figure out is to compile docs locally and then once we have set a direction I can work on this.

**Comment by inoas** [comment]
Yes, I ll leave stdlib be and the next gleam timeslot I should take a look at the docs generator.
